### PR TITLE
Restructure Get Started section

### PR DESCRIPTION
**Pull request by 0xLucca**
This PR removes the Get Started section and relocates its content.

### CODE REVIEW

**Review by nhussein11 — APPROVED** [review]

File: develop/parachains/customize-parachain/index.md [hunk 1]
```diff
@@ -26,9 +26,9 @@ The [FRAME directory](https://github.com/paritytech/polkadot-sdk/tree/master/sub
     </a>
   </div>
     <div class="card">
-    <a href="https://paritytech.github.io/polkadot-sdk/master/frame_support/index.html" target="_blank">
+    <a href="https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/frame_runtime/index.html" target="_blank">
       <h2 class="title">FRAME Rust docs</h2>
-      <p class="description">Check out the Rust docs for the <code>frame_support</code> crate to view the support code for the runtime.</p>
+      <p class="description">Check out the Rust docs for the Substrate’s framework of choice to build a runtime, <code>FRAME</code>.</p>
```
Comment (eshaben) [inline develop/parachains/customize-parachain/index.md:31]:
```suggestion
      <p class="description">Check out the Rust documentation for FRAME, Substrate's preferred framework for building runtimes.</p>
```

File: develop/parachains/deployment/index.md [hunk 2]
```diff
@@ -47,7 +47,22 @@ flowchart TD
...
+
+- **Runtime compilation** - Polkadot SDK-based blockchains are built with Wasm, a highly portable and efficient format. Compiling your blockchain's runtime into Wasm ensures it can be executed reliably across various environments, guaranteeing network-wide compatibility and security. The [srtool](https://github.com/paritytech/srtool){target=\_blank} is helpful for this purpose since it allows you to compile [deterministic runtimes](/develop/parachains/deployment/build-deterministic-runtime/){target=\_blank}
+
+- **Generate chain specifications** - the chain spec file defines the structure and configuration of your blockchain. It includes initial node identities, session keys, and other parameters. Defining a well thought-out chain specification ensures that your network will operate smoothly and according to your intended design
```
Comment (eshaben) [inline develop/parachains/deployment/index.md:54]:
```suggestion
- **Generate chain specifications** - the chain spec file defines the structure and configuration of your blockchain. It includes initial node identities, session keys, and other parameters. Defining a well-thought-out chain specification ensures that your network will operate smoothly and according to your intended design
```

File: develop/parachains/deployment/index.md [hunk 3]
```diff
@@ -47,7 +47,22 @@ flowchart TD
     style group3 fill:#ffffff,stroke:#6e7391,stroke-width:1px
 ```
 
-For more details, check out the [Deploy a Parachain to Polkadot](/develop/parachains/get-started/deploy-parachain-to-polkadot/) overview.
+- **Local development and testing** - the process begins with local development and testing. Developers focus on building the runtime by selecting and configuring the necessary pallets while refining network features. In this phase, it's essential to run a local TestNet to verify transactions and ensure the blockchain behaves as expected. Unit and integration tests are also crucial for ensuring the network works as expected before launch. Thorough testing is conducted, not only for individual components but also for interactions between pallets
```
Comment (eshaben) [inline develop/parachains/deployment/index.md:50]:
```suggestion
- **Local development and testing** - the process begins with local development and testing. Developers focus on building the runtime by selecting and configuring the necessary pallets while refining network features. In this phase, running a local TestNet is essential to verify transactions and ensure the blockchain behaves as expected. Unit and integration tests ensure the network works as expected before launch. Thorough testing is conducted, not only for individual components but also for interactions between pallets
```

**Review by eshaben — COMMENTED** [review]
Sweet! Just a couple minor things I noticed (that were probably there before haha 😅)

**Comment by 0xLucca** [comment]
Suggestions and merge conflicts fixed. Should be ready to merge @eshaben

**Review by eshaben — APPROVED** [review]
LGTM!

**Pull request merged**